.github/workflows/build.yml

```diff
@@ -181,6 +181,7 @@ jobs:
           run: |
             echo "::add-matcher::sources/nuttx/.github/gcc.json"
             cd sources/testing
+            echo -e "/avr\n-avr32dev1:nsh\n-avr32dev1:ostest" >> testlist/avr-mips-riscv-x86-xtensa.dat
```

Comment:
This will be removed once the CI passes.  This should force building of the AVR targets